fish-rust/src/fd_monitor.rs

```diff
@@ -11,8 +11,8 @@ use crate::ffi::void_ptr;
 use crate::flog::FLOG;
 use crate::threads::assert_is_background_thread;
 use crate::wutil::perror;
-use errno::errno;
-use libc::{self, c_void, EAGAIN, EINTR, EWOULDBLOCK};
+use libc::{self, c_void};
```

Comment:
(if we take this, should squash all the "Port" commits into one, and add a bit more info to the commit message)

Reply:
Sure, this was done to make it easy to drop commits as needed, but I don't hear any pushback related to specific modules so I will start squashing stuff down to a more reasonable amount.